### DISCUSSION TITLE
dexie-export-import as script ?

### DISCUSSION THREAD

**Discussion by ledahu** [dexie/Dexie.js#1346] [Q&A] [answered]
Hi

im using Dexie as script, not a module 

and now i need the addon dexie-export-import...

is it a way ? 

thank you

**Comment by dfahlander** [comment, marked as answer]
Yes you can just add it as a plain script tag after adding dexie.

**Reply by jcoonrod** [reply]
Thanks - I have tried that, but it shows that importDB is not defined. Can you point me to a working example?

**Reply by dfahlander** [reply]
I see. When loading the addon using a plain script tag (and not as a module), the explicit exports are not available. Instead, use `Dexie.import()`, `db.export()` or `db.import()` as [described on this page](https://dexie.org/docs/ExportImport/dexie-export-import) as alternatives to importDB(), exportDB() and importInto().
